config/environments/test.rb

```diff
@@ -45,4 +45,5 @@
 
   # Raises error for missing translations.
   # config.action_view.raise_on_missing_translations = true
+  config.hosts << "www.example.com"
```

Comment:
What is this doing? I'm not familiar with this.

Reply:
this is just for testing so we can hit endpoints using get `api/v1/lalala`